src/chordsymbol.ts

```diff
@@ -409,26 +409,12 @@ export class ChordSymbol extends Modifier {
 
   constructor() {
     super();
-    this.resetFont();
-  }
-
-  /**
-   * Default text font.
-   * Choose a font family that works well with the current music engraving font.
-   * @override `Element.TEXT_FONT`.
-   */
-  static get TEXT_FONT(): Required<FontInfo> {
-    let family = 'Roboto Slab, Times, serif';
     if (Tables.currentMusicFont().getName() === 'Petaluma') {
       // Fixes Issue #1180
-      family = 'PetalumaScript, Arial, sans-serif';
+      this.textFont.family = 'PetalumaScript, Arial, sans-serif';
```

Comment:
Should this alternate font family be stored in CommonMetrics as well? We might as well put them all in the same place instead of having `PetalumaScript, Arial, sans-serif` in a separate file.

Reply:
This is resolved in #15